content/id/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Controller
+id: controller
+date: 2018-04-12
+full_link: /docs/concepts/architecture/controller/
+short_description: >
+   Kontrol tertutup yang mengawasi kondisi bersama dari klaster melalui
```

Comment:
line 7-9 must have 2 leading spaces, not 3, this is the reason of breaking

Reply:
It is done! 
Also, I realized that now the newline allowed in the "short description". It should be a single line.